dotnetv3/Lambda/ListFunctionsExample/ListFunctionsExample/ListFunctions.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+namespace ListFunctionsExample
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using Amazon;
+    using Amazon.Lambda;
+    using Amazon.Lambda.Model;
+
+    /// <summary>
+    /// This example shows two ways to list the AWS Lambda functions you have
+    /// created for your account. It will only list the functions within one
+    /// AWS Region at a time, however, so you need to pass the AWS Region you
```

Comment:
You don't have to say "AWS Region" after the first use, just "Region", but it's not something you need to change.

Reply:
I'll keep that in mind in the future. Thanks!